package.json

```diff
@@ -41,7 +41,7 @@
     "node-progress": "^0.1.0",
     "requestretry": "^2.0.2",
     "semantic-release": "^15.13.12",
-    "sharp": "^0.20.8",
+    "sharp": "0.20.5",
```

Comment:
is this for the drone issue we hit earlier?

Reply:
Pinned it because 0.20.8 requires image libraries that don't exist on ubuntu (or at least I wasn't able to find them). Specifically it's libwebp.so.7

Reply:
ah, interesting